biorxiv/article_distances/biorxiv_article_distances.py

```diff
@@ -0,0 +1,499 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Find published articles missing from bioRxiv
```

Comment:
Just to clarify you are trying to link preprint dois with published doi. What is the current way they are being linked? And so there is a problem with some links not existing that should?

Reply:
>Just to clarify you are trying to link preprint dois with published doi.

Correct. I'm trying to link preprints with their missing publish dois

>What is the current way they are being linked?

Great question. I don't know how biorxiv currently links preprints with published dois. I'd speculate they have their own custom search procedure.

>And so there is a problem with some links not existing that should?

Correct. There are preprints that are missing their published version's doi. If I can establish these links it is another bonus for the manuscript as well as gives biorxiv another way to keep track of preprints.